tests/HTML5ValueTest.php

```diff
@@ -73,4 +75,22 @@ public function testMixedNewlines()
             'Newlines get converted'
         );
     }
+
+    public function testShortcodeValue()
+    {
+        ShortcodeParser::get('default')->register(
+            'test_shortcode',
+            function () {
+                return 'bit of test shortcode output';
+            }
+        );
+        $content = DBHTMLText::create('Test', ['shortcodes'=>true])
```

Comment:
the `=>` needs spaces around it.

Reply:
https://travis-ci.org/silverstripe/silverstripe-html5/jobs/355285299#L628 :P

Yeah, you're probably right :) #21